src/Proj4.jl

```diff
@@ -34,7 +34,7 @@ end
 
 "Get human readable error string from proj.4 error code"
 function _strerrno(code::Cint)
-    bytestring(ccall((:pj_strerrno, libproj), Cstring, (Cint,), code))
+    bytestring(ccall((:pj_strerrno, libproj), Ptr{Cchar}, (Cint,), code))
```

Comment:
Good work with the v0.3 stuff.

I'd suggest to leave this as `Cstring` - that's available in `Compat.jl`, or am I mistaken?